cyoa/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,44 @@
+class UsersController < ApplicationController
+  def index
+    @hi_score_games = Game.high_scores
+    @users = @hi_score_games.map(&:user)
```

Comment:
This is an `n+1` problem because it loads each user. Best to eager load it: `Game.high_scores.include(:user)`.